pkg/apis/cnsoperator/config/cns.vmware.com_storagepolicyusages.yaml

```diff
@@ -106,4 +106,106 @@ spec:
             type: object
         type: object
     served: true
+    storage: false
```

Comment:
do we need to add status subresource for v1alpha1 version of StoragePolicyUsage  ?

Reply:
no we are not adding subresource in v1alpha1

Reply:
That means it is not backward compatible?  What's the upgrade impact if there are already objects created with v1alpha1 API?

Reply:
added logs of testing performed with storagepolicyusage version v1alpha1